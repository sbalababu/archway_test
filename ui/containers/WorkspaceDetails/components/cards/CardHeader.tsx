import * as React from 'react';

interface Props {
  children: React.ReactNode;
}

const CardHeader = ({ children }: Props) => (
  <div
    style={{
      fontSize: 18,
      marginBottom: 16,
      textTransform: 'uppercase',
      display: 'flex',
      alignItems: 'center',
    }}
  >
    {children}
  </div>
);

export default CardHeader;
