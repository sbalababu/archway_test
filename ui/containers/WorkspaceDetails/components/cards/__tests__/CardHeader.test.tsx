import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import CardHeader from '../CardHeader';

describe('CardHeader', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<CardHeader children={null} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
