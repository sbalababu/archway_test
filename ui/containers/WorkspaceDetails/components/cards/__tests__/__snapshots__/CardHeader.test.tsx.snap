// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardHeader renders correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "fontSize": 18,
      "marginBottom": 16,
      "textTransform": "uppercase",
    }
  }
/>
`;
